monitor/src/main/java/edu/asu/diging/monitor/web/admin/AddRecipientController.java

```diff
@@ -22,10 +25,23 @@
 	@Autowired
 	private INotificationManager manager;
 	
+	@RequestMapping(value="/admin/recipients/show", method=RequestMethod.GET)
```

Comment:
this should be in its own controller (it has nothing to do with adding recipients); also, let's just make it `/admin/recipients` or `/admin/recipient/list`